### PR TITLE
usermanual : iop layouts

### PR DESCRIPTION
**Pull request by AlicVB**
Documentation for the new iop layout presets/manager

Keep in mind that I'm not really good at writing English, so proofreading is needed here ;)

### CODE REVIEW

**Comment by TurboGit** [comment]
@AlicVB : I'll review, I'm not terrible in English either, but you may know that there is an ongoing project to rework the whole dt documentation. So at the end it does not matter too much :)

**Comment by AlicVB** [comment]
Ok, I've also completely reordered the modules part to present them like the default preset.
Not too much to proofread, but maybe some double-check needed, to verify I've not missed or inverted something ;)

When doing this work, I've found that basic adjustments isn't described... @TurboGit : should I open an issue for that ?

**Comment by AlicVB** [comment]
a note for @aurelienpierre and any other proofreader : 
the description of the three groups are copy-paste from PR #5322 . Maybe we should enhance them a bit, especially for the third one ?

**Comment by TurboGit** [comment]
> When doing this work, I've found that basic adjustments isn't described... @TurboGit : should I open an issue for that ?

Yes please !

File: doc/usermanual/darkroom/modules/effects/group_effects.xml [hunk 1]
```diff
@@ -1,35 +1,44 @@
-<sect2 status="final" id="effect_group">
+<sect2 status="final" id="effects_group">
```
Comment (TurboGit) [inline doc/usermanual/darkroom/modules/effects/group_effects.xml:1]: Don't we need to update the integrated help (when clicking on ?) as links have changed? Probably true for many other changes here I think.

Comment (AlicVB) [reply doc/usermanual/darkroom/modules/effects/group_effects.xml:1]: oh... forgot that... But yes, there should be many links to change (at least all the iops...) I'll have a look !

Comment (AlicVB) [reply doc/usermanual/darkroom/modules/effects/group_effects.xml:1]: hmm... so it seems I have *tons* of links to change in `usermanual_url.c ` (basically each iop link as changed)
problem : once they are changed, they will point to invalid urls until we release the manual... should I do this anyway ? or should we postponed it ?

Comment (AlicVB) [reply doc/usermanual/darkroom/modules/effects/group_effects.xml:1]: done. and in the same time I've fixed lot of invalid urls or missing ones. Surprising that no user raise issues... Maybe a sign that this feature is underutilized ?

**Review by TurboGit — APPROVED** [review]
Good to me, thanks !

**Pull request merged**